app/controllers/bestuurseenheid/zitting/uittreksels/index.js

```diff
@@ -7,6 +7,10 @@ export default class BestuurseenheidZittingUittrekselsIndexController extends Co
     return this.model;
   }
 
+  getBesluit(uittreksel) {
+    return uittreksel.behandelingVanAgendapunt.get('besluiten').firstObject;
```

Comment:
The whole point of using this was to not use the .firstObject because it's deprecated, use [0] instead :).
Also try to avoid using the .get('besluiten') as it's also deprecated, you might need include them in the store query of the page if you are getting an error that behandelingVanAgendapunt is a proxy